include/arch/arm/aarch64/timer.h

```diff
@@ -45,6 +46,22 @@ static ALWAYS_INLINE void arm_arch_timer_enable(unsigned char enable)
 			 : : "r" (cntv_ctl) : "memory");
 }
 
+static ALWAYS_INLINE void arm_arch_timer_irq_mask(unsigned char mask)
```

Comment:
Boolean-valued params should use the `bool` type
```suggestion
static ALWAYS_INLINE void arm_arch_timer_irq_mask(bool mask)
```